src/rt.rs

```diff
@@ -48,14 +51,17 @@ pub(crate) struct Rt<'a> {
     /// Optional handle to a host's software. When software finishes, the handle is
     /// consumed to check for error, which is propagated up to fail the simulation.
     handle: Option<JoinHandle<Result>>,
+
+    /// Configuration of simulation
+    sim_cfg: Config,
```

Comment:
If you switch to just the enable_io boolean here you don't need to worry about a default cfg for the no software case.

Reply:
I don't see this in the recent changes.